tests/conftest.py

```diff
@@ -30,12 +30,14 @@
 
 
 @pytest_asyncio.fixture
-async def app() -> AsyncIterator[FastAPI]:
+async def app(monkeypatch: pytest.MonkeyPatch) -> AsyncIterator[FastAPI]:
     """Return a configured test application.
 
     Wraps the application in a lifespan manager so that startup and shutdown
     events are sent during test execution.
     """
+    monkeypatch.setenv("METRICS_ENABLED", "false")
+    monkeypatch.setenv("METRICS_MOCK", "true")
```

Comment:
Do you know if these setenvs are used, or are the "real" environment variables coming via the tox environment? I thought that these environment variable patches would be too late because the `config` would have already been resolved at this point?

Reply:
Oops, yep, these are leftover from my initial attempt.